### PR TITLE
update timeouts

### PR DESCRIPTION
**Pull request by carltonsmith**
### This pull request changes...

Updates timeouts to match system requirements

### CODE REVIEW

File: tdrs-backend/tdpservice/settings/development.py [hunk 1]
```diff
@@ -49,6 +49,3 @@ class Development(Common):
         "all_applications": True,
         "group_models": True,
     }
-
-    # Session Timeout
-    SESSION_TIMEOUT = 5
```
Comment (carltonsmith) [inline tdrs-backend/tdpservice/settings/development.py:54]: Removing this sets the development `SESSION_TIMEOUT` to the setting in `common.py` which is 30

File: tdrs-backend/tdpservice/settings/local.py [hunk 2]
```diff
@@ -22,6 +22,3 @@ class Local(Common):
     EMAIL_HOST = "localhost"
     EMAIL_PORT = 1025
     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
-
-    # Session Timeout
-    SESSION_TIMEOUT = 5
```
Comment (carltonsmith) [inline tdrs-backend/tdpservice/settings/local.py:27]: Removing this sets the local `SESSION_TIMEOUT` to the setting in `common.py` which is 30

File: tdrs-frontend/.env.development [hunk 3]
```diff
@@ -1,7 +1,7 @@
 REACT_APP_BACKEND_URL=http://localhost:8080/v1
-REACT_APP_TIMEOUT_TIME=6000
-REACT_APP_LOGOUT_TIME=10000
-REACT_APP_DEBOUNCE_TIME=1000
+REACT_APP_TIMEOUT_TIME=1000 * 60 * 20
```
Comment (hilvitzs) [inline tdrs-frontend/.env.development:2]: Timeout time will be 20 minutes

File: tdrs-frontend/.env.development [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+REACT_APP_LOGOUT_TIME=1000 * 60 * 3
```
Comment (hilvitzs) [inline tdrs-frontend/.env.development:3]: Once the modal pops up, the user has three minutes to make a selection before being automatically logged out.

File: tdrs-frontend/.env.development [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
+REACT_APP_DEBOUNCE_TIME=1000 * 30
```
Comment (hilvitzs) [inline tdrs-frontend/.env.development:4]: Debounce time will be 30 seconds

**Review by hilvitzs — APPROVED** [review]

**Pull request merged**